lib/rpc/tls.go

```diff
@@ -194,6 +194,7 @@ func ClientCredentials(secretsDir string) (credentials.TransportCredentials, err
 		return nil, trace.BadParameter("failed to add CA to pool")
 	}
 
+	//nolint:gosec // TODO: set MinVersion
```

Comment:
these ones are probably more important, even if not part of this PR I would probably set atleast TLS 1.2 and probably TLS1.3 soon.